lib/srv/regular/sshserver_test.go

```diff
@@ -1401,13 +1402,55 @@ func TestClientDisconnect(t *testing.T) {
 	require.NoError(t, clt.Close())
 }
 
+func testHelperGetWaitForServerStart(t *testing.T, s *Server, config *ssh.ClientConfig) func() bool {
```

Comment:
Nit: This would be a more "customary" name for this function.
```suggestion
func getWaitForServerStartFunc(t *testing.T, s *Server, config *ssh.ClientConfig) func() bool {
```